blas/testblas/level1double.go

```diff
@@ -1971,6 +2100,15 @@ func DrotmgTest(t *testing.T, d Drotmger) {
 		if !dTolEqual(rx1, test.Rx1) {
 			t.Errorf("drotmg rx1 mismatch %v: expected %v, found %v", test.Name, test.Rx1, rx1)
 		}
+
+		y := []float64{test.Y1}
```

Comment:
Sorry, I'm confused. Isn't this a test of `Drotm` and not `Drotmg`? As far as I can tell this only uses data from Drotm

Reply:
Nope, I'm an idiot, sorry.